src/styles/index.scss

```diff
@@ -1,3 +1,164 @@
+
 body {
   margin: 0;
+  font-family: roboto , helvetica , sans-serif;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px;
+  }
+
+  &__title {
+    display: flex;
+    width: 166px;
+    flex-direction: column;
+    flex-shrink: 0;
+    margin: 0;
+  }
+
+  &__title-title {
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    margin-top: 8px;
+    color: $black;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    color: $gray;
+    width: 166px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__reviews-stars {
+    width: 96px;
+    height: 16px;
+  }
+
+  &__reviews-counter {
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $black;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $gray;
+    margin: 0;
+
+  }
+
+  &__price-sum {
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $black;
+    margin: 0;
+
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $blue;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: $white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    margin: 16px;
+  }
+
+  &__button:hover {
+    background-color: $white;
+    color: $blue;
+    border: solid 1px $blue;
+    width: 164px;
+    height: 38px;
+  }
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__stars-item {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 0;
+}
+
+.card__stars--1 .card__stars-item:nth-child(-n+1) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__stars--2 .card__stars-item:nth-child(-n+2) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__stars--3 .card__stars-item:nth-child(-n+3) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__stars--4 .card__stars-item:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.card__stars--5 .card__stars-item:nth-child(-n+5) {
+  background-image: url(../images/star-active.svg);
 }
```

Comment:
it seems like you're doing a lot of repetetive work, please try to stack those properties in 1 selector